features/search_by_implication_both.feature

```diff
@@ -69,9 +69,9 @@ Feature: Search with Implication Both
 
   Scenario: Visitor searches Implication Both with all Demographic Properties
     When I go to the Syntactic Structures search page
```

Comment:
In this example, @dej611 , at this point : the checkbox "Ling" "Linglet" are hidden. If you try to choose them it raise an exception. You have to choose "Both" checkbox before.